wcbench.sh

```diff
@@ -27,13 +27,13 @@ KARAF_SHELL_PORT=8101  # Port that the Karaf shell listens on
 CONTROLLER="OpenDaylight"  # Currently only support ODL
 CONTROLLER_IP="localhost"  # Change this to remote IP if running on two systems
 CONTROLLER_PORT=6633  # Default port for OpenDaylight
-SSH_HOSTNAME="cbenchc"  # You'll need to update this to reflect ~/.ssh/config
```

Comment:
I changed few things while experimenting in my lappy....Must have uploaded the same accidentally..LIke SR3 version of Helium...Will edit those..sorry:(